package.json

```diff
@@ -14,6 +14,7 @@
     "start:frontend": "npm run start --prefix frontend",
     "start:dev:frontend": "npm run start:dev --prefix frontend",
     "start:prod:backend": "npm run start:prod --prefix backend",
+    "start:e2e": "concurrently --kill-others \"npm run start:backend\" \"npm run start:dev:frontend\"",
```

Comment:
run this command before running starting the e2e tests. it is needed because currently, the `prisma migrate reset --force` command restarts the dev server every time which causes problems